src/questionnaireitem_media_video.js

```diff
@@ -20,6 +20,8 @@ function QuestionnaireItemMediaVideo(className, question, required, url, readyOn
 
     TheFragebogen.logger.debug(this.constructor.name + "()", "Set: className as " + this.className + ", urls as " + this.height + ", width as " + this.width);
 
+    this.videoSource = null;
```

Comment:
What is the advantage of keeping track of the video sources?
I think removing the node from DOM should remove the sources and clear their memory, or?
If not so, please provide as a separate PR.